actioncable/lib/action_cable/channel/streams_handlers/base.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module StreamsHandlers
+      class Base # :nodoc:
```

Comment:
Can we move `Base`'s `:nodoc:` declaration up to `StreamHandlers`? Same deal with `Custom`.